docs/wiki/get-started/IO-with-Adapters.md

```diff
@@ -0,0 +1,65 @@
+In [First Steps](First-Steps) and [More with csp](More-with-CSP) you created example/sample data for the streaming workflows. In real workflows, you use data stored in particular formats and storage spaces, that can be accessed directly or through an API.
+
+csp has [several built-in "adapters" to access certain types of data](Input-Output-Adapters-API), including Kafka and Parquet. csp requires friendly (Time Series) data types, and the I/O adapters form the interface between the data types. You can also write your own adapters for other data types, check the corresponding how-to guides for more details.
```

Comment:
this link seems broken

Reply:
I'd expect this to not work using only README, but it's accurate when viewed on the wiki: https://github.com/pavithraes/csp/wiki/IO-with-Adapters